pysatIncubator/methods/empirical.py

```diff
@@ -0,0 +1,471 @@
+# -*- coding: utf-8 -*-
+"""Provides default routines for projecting pyglow model values onto locations
+from pysat instruments.
+
+"""
+
+import pandas as pds
+import numpy as np
+import warnings
+
+try:
+    from pyglow.pyglow import Point
+except ImportError:
+    pass
```

Comment:
I think it would be good to throw a logger info message here, so a user isn't surprised if they don't have pyglow.

Reply:
But it looks like you throw a warning elsewhere, so your call.

Reply:
The intent is to warn people only when then try to do something with pyglow, rather than load the whole package.

Reply:
adding comment here